package.json

```diff
@@ -17,6 +17,11 @@
       "es2015"
     ]
   },
+  "eslintConfig": {
```

Comment:
@MikeGertrudes , nice. I was trying to figure out the eslint config in my local, but I ended up with a nested, ugly looking config object for this `package.json`. Can we consider upgrading to `es2016`?  (Would like to experiment Decorator when this code base supports `es2016`)